apprise/plugins/NotifyPushover.py

```diff
@@ -585,10 +588,18 @@ def parse_url(url):
                 NotifyPushover.unquote(results['qsd']['sound'])
 
         # Get the supplementary url
-        if 'url' in results['qsd'] and len(results['qsd']['url']):
-            results['url'] = NotifyPushover.unquote(results['qsd']['url'])
-        if 'url_title' in results['qsd'] and len(results['qsd']['url_title']):
-            results['url_title'] = results['qsd']['url_title']
+        if 'supplemental_url' in results['qsd'] and len(
+            results['qsd']['supplemental_url']
+        ):
+            results['supplemental_url'] = NotifyPushover.unquote(
+                results['qsd']['supplemental_url']
+            )
+        if 'supplemental_url_title' in results['qsd'] and len(
+            results['qsd']['supplemental_url_title']
+        ):
+            results['supplemental_url_title'] = results['qsd'][
+                'supplemental_url_title'
+            ]
```

Comment:
Yeah, I know this whole section doesn't look great, especially these lines... this was the suggestion `black` gave me for 79 characters to a line for this 🤷 

I'm happy to call it something besides `supplemental_url`/`supplemental_url_title` but not sure what else to use.